Configurability/ConfigurableServiceHelper.php

```diff
@@ -224,6 +232,43 @@ public function validate(array $stepConfig, array &$context)
         }
     }
 
+    /**
+     * Nasty hack to be able to easily display context variables in producers.
+     *
+     * @param array $actions
+     * @param $context
+     */
+    protected function debug($actions, &$context)
+    {
+        if (!is_array($actions)) {
+            return;
+        }
+        foreach ($actions as $action => $parameters) {
+            if ('display' == $action) {
+                foreach ($parameters as $var) {
+                    echo "\n";
+                    echo $var.' => ';
+                    $command = 'print_r($'.$var.');';
+                    eval($command);
+                }
+                flush();
+                ob_flush();
+            }
+
+            if ('sleep' == $action) {
+                echo 'Sleeping for '.$parameters.' seconds due to sleep step...'."\n";
+                sleep($parameters);
+            }
+
+            if ('exit' == $action && false !== $parameters) {
+                echo 'Exit due to debug step'."\n";
+                exit;
+            }
```

Comment:
could also add an action to do something like this
https://github.com/smartboxgroup/integration-framework-bundle/blob/master/Core/Processors/Processor.php#L140

Reply:
Done, great idea.